src/Microsoft.OData.Client/IBaseEntityType.cs

```diff
@@ -15,7 +15,7 @@ public interface IBaseEntityType
         /// <summary>
         /// DataServiceContext for query provider.
         /// </summary>
-        DataServiceContext Context { get; set; }
+        DataServiceContext EntityTrackingContext { get; set; }

```

Comment:
I don't think we should name this property `EntityTrackingContext` maybe `ServiceContext` or `DataServiceContext` could do.

Reply:
Also. there are no tests that need to be updated by this change?

Reply:
I have renamed the property to `DataServiceContext`.